src/ts/ContentMetadataBuilder.ts

```diff
@@ -129,15 +140,8 @@ export class ContentMetadataBuilder {
     this.metadata.applicationName = newValue;
   }
 
-  set custom(newValue: Metadata['custom']) {
-    this.metadata.custom = newValue;
-  }
-
-  get custom(): Metadata['custom'] {
-    return {
-      ...this.metadataOverrides.custom,
-      ...this.metadata.custom, // Keep our custom tags in case someone tries to override them
-    };
+  addToCustom(toAdd: Metadata['custom']) {
```

Comment:
Why this API change? 🤔 

Previously, the customer could do this easily themselves, but also decide to remove a key. I don't think this is still possible. But to be fair, I'm not sure if there's a use case for that or if Conviva likes that.

Reply:
It is not exposed to customers, this is our internal API only.

Reply:
This is what is exposed to customers https://github.com/bitmovin/bitmovin-player-web-samples/blob/main/react/bundled-player/package-based-with-conviva/src/App.tsx#L81.

Reply:
Yes, of course 🤦 I think my caffeine level is still too low this morning 😅